CHANGELOG.md

```diff
@@ -2,6 +2,8 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 
 ### [Unreleased](https://github.com/cucumber/cucumber-js/compare/v3.2.1...master) (In Git)
 
+* added "asyncawait" option to the snippetInterface formatting option
```

Comment:
How about naming this `async-await`?

Reply:
I don't like the idea of putting a "-" character in a CLI option. But, if you prefer to do that - let me know, and I will.

Reply:
This isn't a typical cli option: https://github.com/cucumber/cucumber-js/blob/master/docs/cli.md#interface. I'd prefer the dash to make it easier to read.